tools/syz-declextract/.clang-format

```diff
@@ -0,0 +1,262 @@
+---
```

Comment:
We have older clang-format on CI which does not understand some of these arguments.
But also it's better to use some standard predefined style + few tweaks, see e.g.:
https://github.com/google/syzkaller/blob/master/.clang-format
maybe something like:
```
BasedOnStyle: LLVM
ColumnLimit: 120
```

Reply:
Done